### ISSUE TITLE
Missing or Inconsistent spacing between words

### ISSUE THREAD

**Issue by ArmorBearer** [helmadik/LSJLogeion#6]
I have found many instances of spaces that appear to be missing between words, yet this is not always the case.  For example, consider the entry for 'ἃ ἃ'.  The following XML sequence is in the file:

`<quote lang="greek">ἃ ἃ δασυνθὲν γέλωτα δηλοῖ</quote><author>Hsch.</author>`

For this, we display:

`δηλοῖHsch.`

This is because the XML has no space between the `<quote>` and `<author>` elements.  Yet, in other places in the same entry, the space is present:

`<author>Pl.Com.</author> 16</bibl> (prob. l.), etc.; <cit>...`

Here we see a space after `</author>` and `</bibl>` and before `<cit>`.

Also in this example from the entry for 'ἀάβακτοι':

`<quote lang="greek">-βυκτον</quote><author>Cyr.</author>`

For this, we display:

`"-βυκτονCyr."`

I mention this here to benefit others, rather than to request a change in the data, because adding the spaces would be a huge task, and it actually wouldn't benefit us.  I have found after playing around with programmatic changes that the Microsoft .NET implementation of XML support will strip out spaces between elements if there are no other characters!

It may be possible to address this through the use of CSS.

**Comment by helmadik** [comment]
Yes, this is really annoying! Over time I've added many spaces, but this is another good case. It doesn't fit the usual format: If a quote is attributed, it ought to be `<cit><quote...  **</quote>SPACE<bibl.... >**<author...  <title... </bibl></cit>`

PS I went back and checked, and found a single instance of `</quote><author>`, which I've corrected. I'd either recently added the space, or your software has been eating more of them.. BTW there are many cases of  `</quote>SPACE<author>` for all the references that Perseus could not give a proper machine citation for.

**Comment by ArmorBearer** [comment]
Additional examples of spacing where punctuation is involved:

`Krumbacher, in one of his latest articles in the <title>Byzantinische Zeitschrift</title>,<note n="2" anchored="yes" place="unspecified">xviii (1909), 708 ff.</note>; criticized the project...`

If the `<note>` is shown, the display will show "Byzantinische Zeitschrift,xviii (1909), 708 ff.; criticized...", without a space between the comma and the Roman numeral.  But if the <note> is hidden, the display will show ",; criticized..."

Also:

`(Sch.<bibl n="Perseus:abo:tlg,0012,002:3:279" default="NO" valid="yes"><title>Od.</title> 3.279</bibl>, etc.`

If the `<title>` is visible, the display will show "Sch.Od. 3.279, etc."

`(cf. <bibl n="Perseus:abo:tlg,0086,025:1022b:32" default="NO" valid="yes"><author>Arist.</author><title>Metaph.</title>`

If the `<title>` is visible, the display will show "Arist.Metaph."

**Comment by helmadik** [comment]
Great. I'll try to track some of those down. Update: Uploading new files this evening.
Do report more; I've spent some time trying to eliminate lots of these but you may well find more problem patterns.
